cluster/ci/openshift-monitoring/prometheus-k8s-rules_prometheusrule.yaml

```diff
@@ -0,0 +1,1454 @@
+          summary: Errors while sending alerts from Prometheus
+        expr: |
+          rate(prometheus_notifications_errors_total{job="prometheus-k8s"}[5m]) / rate(prometheus_notifications_sent_total{job="prometheus-k8s"}[5m]) > 0.03
+        for: 10m
+        labels:
+          severity: critical
+      - alert: PrometheusNotConnectedToAlertmanagers
+        annotations:
+          description: Prometheus {{ $labels.namespace }}/{{ $labels.pod}} is not
+            connected to any Alertmanagers
+          summary: Prometheus is not connected to any Alertmanagers
+        expr: |
+          prometheus_notifications_alertmanagers_discovered{job="prometheus-k8s"} < 1
+        for: 10m
+        labels:
+          severity: warning
+      - alert: PrometheusTSDBReloadsFailing
+        annotations:
+          description: '{{$labels.job}} at {{$labels.instance}} had {{$value | humanize}}
+            reload failures over the last four hours.'
+          summary: Prometheus has issues reloading data blocks from disk
+        expr: |
+          increase(prometheus_tsdb_reloads_failures_total{job="prometheus-k8s"}[2h]) > 0
+        for: 12h
+        labels:
+          severity: warning
+      - alert: PrometheusTSDBCompactionsFailing
+        annotations:
+          description: '{{$labels.job}} at {{$labels.instance}} had {{$value | humanize}}
+            compaction failures over the last four hours.'
+          summary: Prometheus has issues compacting sample blocks
+        expr: |
+          increase(prometheus_tsdb_compactions_failed_total{job="prometheus-k8s"}[2h]) > 0
+        for: 12h
+        labels:
+          severity: warning
+      - alert: PrometheusTSDBWALCorruptions
+        annotations:
+          description: '{{$labels.job}} at {{$labels.instance}} has a corrupted write-ahead
+            log (WAL).'
+          summary: Prometheus write-ahead log is corrupted
+        expr: |
+          tsdb_wal_corruptions_total{job="prometheus-k8s"} > 0
+        for: 4h
+        labels:
+          severity: warning
+      - alert: PrometheusNotIngestingSamples
+        annotations:
+          description: Prometheus {{ $labels.namespace }}/{{ $labels.pod}} isn't ingesting
+            samples.
+          summary: Prometheus isn't ingesting samples
+        expr: |
+          rate(prometheus_tsdb_head_samples_appended_total{job="prometheus-k8s"}[5m]) <= 0
+        for: 10m
+        labels:
+          severity: warning
+      - alert: PrometheusTargetScrapesDuplicate
+        annotations:
+          description: '{{$labels.namespace}}/{{$labels.pod}} has many samples rejected
+            due to duplicate timestamps but different values'
+          summary: Prometheus has many samples rejected
+        expr: |
+          increase(prometheus_target_scrapes_sample_duplicate_timestamp_total{job="prometheus-k8s"}[5m]) > 0
+        for: 10m
+        labels:
+          severity: warning
+    - name: prow.rules
```

Comment:
This is the group added for prow. Before that line, all the group are already exist.